activities/FoodChain.activity/activity/activity.info

```diff
@@ -1,6 +1,15 @@
 [Activity]
 name = FoodChain
 activity_version = 4
+license = ASL 2.0
 bundle_id = org.olpcfrance.FoodChain
 exec = sugar-activity-web
 icon = activity-icon
+summary = FoodChain is a pedagogical game to learn name of animals.
+tags = Science;Game
+repository = https://github.com/llaske/FoodChain
```

Comment:
The `repository` tag information would also need to change when repositories have changed.  
Since, the README.md clearly stated that that repository is not going to be maintained.